bin/stencil-start.js

```diff
@@ -160,6 +160,18 @@ async function startServer() {
         }
     });
 
+    Bs.watch('config.json', (event) => {
+        if (event === 'change') {
+            try {
+                themeConfig.resetVariationSettings();
```

Comment:
Just curious when `themeConfig.resetVariationSettings()` can throw errors?

Reply:
haha, never :) That was left from previous trials using `updateConfig()`